src/main/java/seedu/address/logic/commands/schedule/EditScheduleCommand.java

```diff
@@ -32,39 +33,38 @@
             + ": schedules a client with another session. \n"
             + "Existing values will be overwritten by the input values.\n"
             + "Parameters: "
-            + PREFIX_CLIENT_INDEX + "CLIENT "
-            + PREFIX_SESSION_INDEX + "SESSION "
-            + PREFIX_UPDATED_SESSION_INDEX + "UPDATED SESSION "
+            + PREFIX_CLIENT_INDEX + "CLIENT (must be a positive integer) "
+            + PREFIX_SESSION_INDEX + "SESSION (must be a positive integer) "
+            + "[" + PREFIX_UPDATED_SESSION_INDEX + "UPDATED SESSION] "
+            + "[" + PREFIX_IS_PAID + "IS PAID]\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_CLIENT_INDEX + "1 "
             + PREFIX_SESSION_INDEX + "1 "
-            + PREFIX_UPDATED_SESSION_INDEX + "1 ";
+            + PREFIX_UPDATED_SESSION_INDEX + "1 "
+            + PREFIX_IS_PAID + "true ";
 
-    public static final String MESSAGE_EDIT_SCHEDULE_SUCCESS = "Editschedule : \n%1$s";
+    public static final String MESSAGE_EDIT_SCHEDULE_SUCCESS = "Schedule Edited: \n%1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_SCHEDULE = "This Schedule overlaps with an existing Schedule";
+    public static final String MESSAGE_SCHEDULE_NOT_FOUND = "No schedule is associated with this client and session";
 
     private final Index sessionIndex;
     private final Index clientIndex;
```

Comment:
Just to check, so this sessionIndex here only contains the updatedSessionIndex? Because in the EditScheduleDescriptor, there's both sessionIndex and updatedSessionIndex, so it may a bit confusing if sessionIndex is referring to updatedSessionIndex here, especially if Bennett is going to implement the other portion

Reply:
sessionIndex is the previous index (index of session to change)
the reason I delete the updatedSessionIndex is because it is already contained in the descriptor (since this is an optional field). also, I'm afraid that if we have updatedSessionIndex in both command and descriptor, we must ensure both of them are the same
@benclmnt any comment?

Reply:
true, but that's why initially I added in session index as well, we always have all 3 parameters so it won't be too confusing to see if its session index or updated session index, but of course we need to update both values which might take up extra work

Reply:
Yup for consistency, we just put anything optional inside the descriptor